packages/nimble-components/src/select/specs/built-in-labels.md

```diff
@@ -0,0 +1,77 @@
+# Built-in Labels for Select and Combobox
+
+## Problem Statement
+
+Most Nimble controls offer built in labels but the select and combobox controls don't, meaning clients have had to create and style labels manually.
+
+Built-in labels will reduce the effort needed for clients to consume these controls, as well as resolving some accessibility issues (see work item links below).
+
+## Links To Relevant Work Items and Reference Material
+
+[Built-in label for select and combobox](https://github.com/ni/nimble/issues/2183)
+
+Related issues:
+
+-   [Resolve label accessibility issues](https://github.com/ni/nimble/issues/94)
+-   [Fix accessibility issues in Angular Example app](https://github.com/ni/nimble/issues/280)
+
+## Implementation / Design
+
+Implementation details apply to both `nimble-select` and `nimble-combobox`, unless otherwise specified.
+
+### API (Slots)
+
+Default Slot:
+
+-   The `nimble-list-option`s in the control are placed in this slot currently.
+-   We will add support for specifying the label as text content in the same slot. For example:
+    ```html
+    <nimble-select>
+        Select Label Text
+        <nimble-list-option>A</nimble-list-option>
+        <nimble-list-option>B</nimble-list-option>
+    </nimble-select>
+    ```
+
+### Templates / Styling
```

Comment:
Might be worth calling out that the label will be dimmed if the control is disabled since I think we've advertised that as one of the benefits of adding built-in labels. A lot of clients have gotten that wrong and forgotten to dim their custom labels.

Reply:
Added